mappings/net/minecraft/client/gl/ShaderProgramParseException.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2973 net/minecraft/client/gl/ShaderParseException
+CLASS net/minecraft/class_2973 net/minecraft/client/gl/ShaderProgramParseException
```

Comment:
I believe this actually has nothing to do with shaders. When I last checked, it was in common jar.

Reply:
Renamed to `InvalidHierarchicalFileException`. Let me know if there is a better name.